src/ansible_runner/config/runner.py

```diff
@@ -270,35 +269,35 @@ def generate_ansible_command(self):
                 extravars_path = '/runner/env/extravars'
             else:
                 extravars_path = self.loader.abspath('env/extravars')
-            exec_list.extend(['-e', '@{}'.format(extravars_path)])
+            exec_list.extend(['-e', f'@{extravars_path}'])
 
         if self.extra_vars:
             if isinstance(self.extra_vars, dict) and self.extra_vars:
                 extra_vars_list = []
                 for k in self.extra_vars:
-                    extra_vars_list.append("\"{}\":{}".format(k, json.dumps(self.extra_vars[k])))
+                    extra_vars_list.append(f"\"{k}\":{json.dumps(self.extra_vars[k])}")
 
                 exec_list.extend(
                     [
                         '-e',
-                        '{%s}' % ','.join(extra_vars_list)
+                        f'{{{",".join(extra_vars_list)}}}'
```

Comment:
```suggestion
                        ",".join(extra_vars_list)
```

Reply:
Your suggestion isn't the same, though. The intent is to pass the extra vars as inline JSON so the surround parens are required here.

Reply:
The triple `{` and `}` do look weird, but the first two are how you send a literal paren character in a f-string. The third is for the interpolated value.

Reply:
TIL